### PR TITLE
balances in character creation

### PR DESCRIPTION
**Pull request by Dorodomki**
## About The Pull Request

for some reason someone buffed the early start of the characters going with +5 and +10 stat points in some areas, this reverts the clearly convenient start for some characters since is not fair.

## Changelog
:cl:
balance: nerfs some stat points in character creation for humans
/:cl:
¡

### CODE REVIEW

File: code/datums/setup_option/backgrounds/ancestry/ancestry_human.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 	restricted_to_species = list(FORM_HUMAN)
 
-	stat_modifiers = list(STAT_COG = 5, STAT_BIO = 10, STAT_MEC = 10) //Pretty bad perk baked into this so
+	stat_modifiers = list(STAT_COG = 5, STAT_BIO = 5, STAT_VIG = 5) //Pretty bad perk baked into this so//pretty unbalanced start for a character so is going with 5 as every other ancestry.
```
Comment (ThePainkiller) [inline code/datums/setup_option/backgrounds/ancestry/ancestry_human.dm:11]:
```suggestion
	stat_modifiers = list(STAT_COG = 5, STAT_BIO = 5, STAT_MEC = 5) //Pretty bad perk baked into this so
```
No combat stats on Sol Born.

File: code/datums/setup_option/backgrounds/thoughts/thoughts.dm [hunk 2]
```diff
@@ -15,7 +15,7 @@
 	Never did you lose your composure in front of others, uttered a singular tear in your entire life, you are the definition of a hard-ass. \
 	But this left you emotionally stunted, unable to feel any empathy for others or being able to relate to simplest emotions besides your own will to survive. \
 	It's understandable a person like you would find problems in socilizing as life has shown that relying on others shortens your lifespan."
-	stat_modifiers = list(STAT_WIL = 10, STAT_ROB = 5, STAT_TGH = 5, STAT_VIG = 5, STAT_COG = -10)
+	stat_modifiers = list(STAT_WIL = 5, STAT_ROB = 5, STAT_TGH = 5, STAT_VIG = 5, STAT_COG = -10)//the willpower is a strong stat and shouldn't be that high on the start of shift
```
Comment (ThePainkiller) [inline code/datums/setup_option/backgrounds/thoughts/thoughts.dm:18]: Where did you get the idea that Willpower is a strong stat? Do you have any idea what it controls? More than half of what it does is gated behind psionics, which are currently completely disabled, and we don't have bears on the map for the growl check either, the ONLY thing they affect are two things: Sanity damage when witnessing deaths and NSA, and the later is literally a lazy thing that was done when Vivification and Anatomy were removed in lieu for this, and it still isn't as busted as you think it is.
Willpower being strong on a literal background that is **designed** to freak out less in tunnels is literally a design choice, it's not like max HP.

Comment (ThePainkiller) [inline code/datums/setup_option/backgrounds/thoughts/thoughts.dm:18]:
```suggestion
	stat_modifiers = list(STAT_WIL = 10, STAT_ROB = 5, STAT_TGH = 5, STAT_VIG = 5, STAT_COG = -10) // High willpower on a perk designed to freak out less at death
```

File: code/datums/setup_option/backgrounds/thoughts/thoughts.dm [hunk 3]
```diff
@@ -35,7 +35,7 @@
 	Merely standing in presence of such unfortunate individuals lifts their spirit right back up allowing them to go on throught with another day.\
 	This idea is rarity among the many disillusioned spacefarers making your idealist stance on life even more refreshing to see.\
 	Keep in mind however that even the most resillient people can snap and that it's not possible to help everyone."
-	stat_modifiers = list(STAT_WIL = 10, STAT_COG = -5, STAT_BIO = -5)
+	stat_modifiers = list(STAT_WIL = 5, STAT_COG = -5, STAT_BIO = -5)
```
Comment (ThePainkiller) [inline code/datums/setup_option/backgrounds/thoughts/thoughts.dm:38]:
```suggestion
	stat_modifiers = list(STAT_WIL = 10, STAT_COG = -5, STAT_BIO = -5)
```
Stats are balanced on the same -10 / +10 mentality unless perks with negative drawbacks are taken into account
Nerfing this unnecessarily makes the perk literally undesireable to even take. At least split it on another positive if you want to nerf its WIL.

**Review by ThePainkiller — CHANGES_REQUESTED** [review]

File: code/datums/setup_option/backgrounds/path/path.dm [hunk 4]
```diff
@@ -22,7 +22,7 @@
 	desc = "You used to work as an EMT either in civilian or military sector. \
 	You have basics of first aid but not much else."
 
-	stat_modifiers = list(STAT_BIO = 15, STAT_TGH = 5, STAT_MEC = -10, STAT_ROB = 5, STAT_VIG = 5)
+	stat_modifiers = list(STAT_BIO = 15, STAT_TGH = 5, STAT_MEC = -10)//the medic path was too unbalanced with the previous stats so is going down, the "combat medic" can be part of the character background and can be "created" using selected stat points, doesn't need to have extra points in these areas if they can spend 5 points to get these points and 10 points more, resulting in an +15 rob/vig+ medic skills.
```
Comment (Trilbyspaceclone) [inline code/datums/setup_option/backgrounds/path/path.dm:25]:
```suggestion
	stat_modifiers = list(STAT_BIO = 15, STAT_TGH = 5, STAT_MEC = -10) 
```

**Pull request merged**